nalgebra-sparse/src/csr.rs

```diff
@@ -170,6 +170,24 @@ impl<T> CsrMatrix<T> {
         Self::try_from_pattern_and_values(pattern, values)
     }
 
+    /// Try to construct a CSR matrix from raw CSR data with unsorted columns.
```

Comment:
I think the documentation should explain the invariants and error conditions like the documentation for `::try_from_csr_data`